src/util-debug.c

```diff
@@ -53,13 +51,27 @@ SCEnumCharMap sc_log_level_map[ ] = {
     { NULL,             -1 }
 };
 
+SCEnumCharMap sc_log_slevel_map[] = {
+    { "Not set",        SC_LOG_NOTSET },
+    { "None",           SC_LOG_NONE },
+    { "E",              SC_LOG_ERROR },
+    { "W",              SC_LOG_WARNING },
+    { "i",              SC_LOG_NOTICE },
+    { "i",              SC_LOG_INFO },
+    { "i",              SC_LOG_PERF },
+    { "i",              SC_LOG_CONFIG },
+    { "d",              SC_LOG_DEBUG },
+    { NULL,             -1 }
```

Comment:
Is there a convention for using caps for high priority errors?  I do find it a bit of an odd-looking inconsistency.

Reply:
It's essentially only using caps for error and warning, which is meant to draw attention.